app/molecule/page.jsx

```diff
@@ -101,6 +102,43 @@ const page = () => {
       </div>
       <LearnCard />
       <MainProductComponent products={products} />
+      <div className="flex flex-col justify-center items-center m-4">
+        <h1 className="font-bold text-lg mb-10">BLOG</h1>
+        <div className="mt-2 flex w-4/5 m-auto justify-center gap-6 h-1/4">
+          <BlogCard
+            imageLink={"/tek-hucreli.png"}
+            imageAlt="Blog görseli"
+            date="29 Haziran 2025"
+            title="React ile Molekül Tasarımı"
+            excerpt="Bu yazıda, atomik tasarım prensiplerine göre nasıl yeniden kullanılabilir bileşenler oluşturabileceğimizi öğreneceğiz."
+            aspectRatio="aspect-[16/16]"
+          />
+          <BlogCard
+            imageLink={"/tek-hucreli.png"}
+            imageAlt="Blog görseli"
+            date="29 Haziran 2025"
+            title="React ile Molekül Tasarımı"
+            excerpt="Bu yazıda, atomik tasarım prensiplerine göre nasıl yeniden kullanılabilir bileşenler oluşturabileceğimizi öğreneceğiz."
+            aspectRatio="aspect-[16/16]"
+          />
+          <BlogCard
+            imageLink={"/tek-hucreli.png"}
+            imageAlt="Blog görseli"
+            date="29 Haziran 2025"
+            title="React ile Molekül Tasarımı"
+            excerpt="Bu yazıda, atomik tasarım prensiplerine göre nasıl yeniden kullanılabilir bileşenler oluşturabileceğimizi öğreneceğiz."
+            aspectRatio="aspect-[16/16]"
+          />
+          <BlogCard
+            imageLink={"/tek-hucreli.png"}
+            imageAlt="Blog görseli"
+            date="29 Haziran 2025"
+            title="React ile Molekül Tasarımı"
+            excerpt="Bu yazıda, atomik tasarım prensiplerine göre nasıl yeniden kullanılabilir bileşenler oluşturabileceğimizi öğreneceğiz."
+            aspectRatio="aspect-[16/16]"
+          />
+        </div>
```

Comment:
Here, data imported from json file could be used. It would be better for future BE adjustments.